### PR TITLE
render markers

### PR DESCRIPTION
**Pull request by mw222rs**
Renders markers, although I'm having problems refactoring markers to separate file and I'm having some deja vu-feeling that I messed around with this a lot last time I user react-google-maps... 😪

### CODE REVIEW

File: src/components/location-map/marker.js [hunk 1]
```diff
@@ -0,0 +1,18 @@
+import React, {PropTypes} from "react";
+import Marker from "react-google-maps/lib/Marker";
+
+const LocationMarker = ({location: {lat, lng}}) => {
+  const position = {lat, lng};
+  console.log(position);
```
Comment (rk222ev) [inline src/components/location-map/marker.js]: Can we remove this line?

File: src/components/location-map/location-map.js [hunk 2]
```diff
@@ -1,5 +1,10 @@
 import React, {Component, PropTypes} from "react";
-import {GoogleMapLoader, GoogleMap} from "react-google-maps/lib";
+import GoogleMapLoader from "react-google-maps/lib/GoogleMapLoader";
+import GoogleMap from "react-google-maps/lib/GoogleMap";
+import Marker from "react-google-maps/lib/Marker";
+import map from "lodash/map";
+
+// import {LocationMarker} from "../../components";
```
Comment (rk222ev) [inline src/components/location-map/location-map.js]: Can we remove this line?

Comment (mw222rs) [inline src/components/location-map/location-map.js]: Done!

File: src/components/location-map/marker.js (hunk 1 above)
Comment (mw222rs) [inline src/components/location-map/marker.js]: Done!

**Pull request merged**